google/cloud/aiplatform/featurestore/entity_type.py

```diff
@@ -1146,6 +1128,129 @@ def ingest_from_gcs(
             request_metadata=request_metadata,
         )
 
+    def ingest_from_df(
+        self,
+        feature_ids: List[str],
+        feature_time: Union[str, datetime.datetime],
+        df_source: "pd.DataFrame",  # noqa: F821 - skip check for undefined name 'pd'
+        feature_source_fields: Optional[Dict[str, str]] = None,
+        entity_id_field: Optional[str] = None,
+        request_metadata: Optional[Sequence[Tuple[str, str]]] = (),
+    ) -> "EntityType":
```

Comment:
Can you add a comment to mention whether this function wait for ingestion complete (in other words, do users expected    data is available once the function return)

Reply:
added